packages/vite/src/client/client.ts

```diff
@@ -254,7 +254,7 @@ export function updateStyle(id: string, content: string): void {
     if (!style) {
       style = new CSSStyleSheet()
       style.replaceSync(content)
-      // @ts-ignore
+      // @ts-expect-error using experiment API
```

Comment:
I think the https://typescript-eslint.io/rules/ban-ts-comment requests for a colon
So if we turn on this rule later, we do not to update your lines

```suggestion
      // @ts-expect-error: using experiment API
```